app/api.py

```diff
@@ -245,22 +245,12 @@ async def financial_aid(profile_id: str):
 
```

Comment:
This looks fantastic! You made quick work of this task, way to go! If we need numeric output we can always create a second function for that.